api/src/env.ts

```diff
@@ -380,37 +381,36 @@ export const getEnv = () => env;
  * When changes have been made during runtime, like in the CLI, we can refresh the env object with
  * the newly created variables
  */
-export function refreshEnv(): void {
+export async function refreshEnv(): Promise<void> {
 	env = {
 		...defaults,
 		...process.env,
-		...processConfiguration(),
+		...(await processConfiguration()),
 	};
 
 	process.env = env;
 
 	env = processValues(env);
 }
 
-function processConfiguration() {
+async function processConfiguration() {
 	const configPath = path.resolve(process.env['CONFIG_PATH'] || defaults['CONFIG_PATH']);
 
 	if (fs.existsSync(configPath) === false) return {};
 
 	const fileExt = path.extname(configPath).toLowerCase();
 
 	if (fileExt === '.js') {
```

Comment:
```suggestion
	if (['.js', '.cjs', '.mjs'].includes(fileExt)) {
```

let's allow for the CJS fallback same as for extensions.